applications/luci-app-apinger/root/usr/share/rpcd/acl.d/luci-app-apinger.json

```diff
@@ -0,0 +1,15 @@
+{
+	"luci-app-apinger" : {
+		"description" : "Grant access to LuCI app Apinger",
+		"read" : {
+			"ubus" : {
+				"apinger" : [ "*" ]
```

Comment:
I think we also need a `read` and `write` uci acl for arpinger uci file

Reply:
Is it really needed? From LuCI, it's not reading/writing to /etc/config/apinger explicitly, instead, Its using RPC to get apinger overview.

Reply:
As I see it, you are using the [uci handling](https://github.com/openwrt/luci/pull/5887/files#diff-43eb48c6a064931b875ac7df92656e5fef64b0d46101ec2d94b68b7984728768R4).
You can create a user and give him only the authorisation for the luci-app-apinger, then we will see if it also works without the uci acl.

Reply:
Yes, You are right. Updated rpcd ACL to handle apinger config.